src/engine/tile/tileset_util.cpp

```diff
@@ -49,15 +53,20 @@ TilesetUnsetBitMaskTile(Tileset &tileset, int mouse_x, int mouse_y)
 
     if (row_tile_clicked < tileset.sprite_sheet.n_rows && col_tile_clicked < tileset.sprite_sheet.n_cols)
     {
-        int tile_location_x = col_tile_clicked * tileset.sprite_sheet.cell_height();
-        int tile_location_y = row_tile_clicked * tileset.sprite_sheet.cell_width();
+        int tile_location_x = col_tile_clicked * tileset.sprite_sheet.cell_width();
+        int tile_location_y = row_tile_clicked * tileset.sprite_sheet.cell_height();
+
+        int bit_cell_size_height = tileset.sprite_sheet.cell_height() / 3;
```

Comment:
put `3` in a variable and name it what it means. Or....... combine `2` and `3` in one variable since i think they are related.